projects/cdk/directives/checked/tests/checked.directive.spec.ts

```diff
@@ -0,0 +1,85 @@
+import {ElementRef, Renderer2} from '@angular/core';
+import {TuiCheckedDirective} from '@taiga-ui/cdk';
+
+describe(`TuiCheckedDirective`, () => {
+    const spyRendererSetProperty = jest.fn();
+    let directive: TuiCheckedDirective;
+    let el: ElementRef<HTMLInputElement>;
+    let renderer: Renderer2;
+
+    class MockRenderer {
```

Comment:
We should probably just drop `Renderer` from that directive. BTW, can we just do `HostBinding` instead?

Reply:
fixed